integrations/gitlab-v2/tests/clients/test_base_client.py

```diff
@@ -147,14 +147,14 @@ async def test_get_group_resource(self, client: GitLabClient) -> None:
             # Act
             results: list[dict[str, Any]] = []
             async for batch in client.get_groups_resource(
-                [group], "issues"
+                [group], "issues", params=None
```

Comment:
```suggestion
                [group], "issues"
```
I don't think you need it